stm/asyncTaskQueue.ml

```diff
@@ -130,19 +130,16 @@ module Make(T : Task) () = struct
         (* Options to discard: 1 argument *)
         | ( "-async-proofs" | "-vio2vo" | "-o"
           | "-load-vernac-source" | "-l" | "-load-vernac-source-verbose" | "-lv"
-          | "-compile" | "-compile-verbose"
           | "-async-proofs-cache" | "-async-proofs-j" | "-async-proofs-tac-j"
           | "-async-proofs-private-flags" | "-async-proofs-tactic-error-resilience"
           | "-async-proofs-command-error-resilience" | "-async-proofs-delegation-threshold"
           | "-async-proofs-worker-priority" | "-worker-id") :: _ :: tl ->
           set_slave_opt tl
         (* We need to pass some options with one argument *)
-        | ( "-I" | "-include" | "-top" | "-topfile" | "-coqlib" | "-exclude-dir" | "-compat"
-          | "-require-import" | "-require-export" | "-require-import-from" | "-require-export-from"
-          | "-ri" | "-re" | "-rifrom" | "-refrom" | "-load-vernac-object"
+        | ( "-I" | "-include" | "-top" | "-topfile" | "-coqlib" | "-exclude-dir"
           | "-w" | "-color" | "-init-file"
-          | "-profile-ltac-cutoff" | "-main-channel" | "-control-channel" | "-mangle-names" | "-set" | "-unset"
-          | "-diffs" | "-mangle-name" | "-dump-glob" | "-bytecode-compiler" | "-native-compiler" as x) :: a :: tl ->
+          | "-profile-ltac-cutoff" | "-main-channel" | "-control-channel"
+          | "-dump-glob" | "-bytecode-compiler" | "-native-compiler" as x) :: a :: tl ->
```

Comment:
You need to add them to "options to discard" not just delete them.

Reply:
OK thanks!

Reply:
Should be better now.